src/pages/terms-and-conditions/_business-grid.tsx

```diff
@@ -36,55 +38,138 @@ const Col = ({ Icon, content, link_title, title, url }: ColProps) => (
     </GridCol>
 )
 
-const BusinessGrid = () => (
-    <StyledContainer>
-        <StyledGrid
-            columns="repeat(3, 1fr)"
-            column_gap="15.4rem"
-            row_gap="8rem"
-            tablet_columns="repeat(2, 1fr)"
-            mobile_columns="1fr"
-            mobile_row_gap="10rem"
-        >
+const PaymentAgentsPdf = () => {
+    if (getLanguage() === 'fr') {
+        return (
             <Col
-                Icon={General}
-                title={localize('General terms of use')}
-                content={localize(
-                    'Terms and ethical standards for all our affiliates, introducing brokers, API users, and payment agents',
-                )}
-                url="/tnc/business-partners-general-terms.pdf"
-                link_title={localize('General terms of use')}
+                Icon={PA}
+                title={localize('Payment agents')}
+                content={localize('Additional terms for our payment agents')}
+                url="/tnc/business-partners-affiliates-and-introducing-brokers-french.pdf"
+                link_title={localize('Payment agents')}
+            />
+        )
+    }
+
+    if (getLanguage() === 'id') {
```

Comment:
Hi @konstantinosG-deriv 
Could you please rename the pdf files and use this clean version?

```
const PaymentAgentsPdf = () => {
```suggestion
    if (getLanguage() === 'id') {
```
    const language = getLanguage();
    const url = `/tnc/business-partners-affiliates-and-introducing-brokers-${language === 'en' ? 'english' : language}.pdf`;

    return (
        <Col
            Icon={PA}
            title={localize('Payment agents')}
            content={localize('Additional terms for our payment agents')}
            url={url}
            link_title={localize('Payment agents')}
        />
    );
}
```

Reply:
@habib-deriv can we add fallback or default value as well? Mentioning some examples blow.
 
 Example 1
 
`/business-partners-affiliates-and-introducing-brokers-${language === 'en' ? 'english' : (language || 'english')}.pdf` 

Example 2

`/business-partners-affiliates-and-introducing-brokers-${language || 'en'}.pdf`

@konstantinosG-deriv

Reply:
Yeah, Nice catch @fasihali-deriv  I think we don't have pdf for some languages, a fallback is needed

Reply:
Nice, thanks for the suggestions 👍 

For the fallback statement the ternary statement it will return the language always since the value exists in the **getLanguage().** . Have to modify it a bit because for example if language is set to **de** then the statement will return **de** and there is no pdf file for German language.